runtime/statemine/src/lib.rs

```diff
@@ -494,6 +496,8 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 
 impl parachain_info::Config for Runtime {}
 
+impl cumulus_pallet_aura_ext::Config for Runtime {}
```

Comment:
don't we put this into `construct_runtime!`?

Reply:
Yes. It is there as : `AuraExt: cumulus_pallet_aura_ext::{Pallet, Config},`